fe/src/app/list/page.tsx

```diff
@@ -0,0 +1,50 @@
+import AdvertCard from "@/components/AdvertCard/AdvertCard";
+import { Breadcrumbs, Typography } from "@/lib/materialTailwindExports";
+import { UserAdvertType } from "@/types/AdvertType";
+import { BASE_API_URL, USER_ADVERTS_ROUTE } from "@/utils/urls";
+import { getAccessToken } from "@auth0/nextjs-auth0";
+import Link from "next/link";
+
+const getUserAdvertsData = async () => {
+  const { accessToken } = await getAccessToken();
+  if (!accessToken) {
+    throw new Error(`Requires authorization`);
+  }
+  const res = await fetch(
+    `${BASE_API_URL}${USER_ADVERTS_ROUTE}`,
+    {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    }
+  );
+  if (!res.ok) {
+    return null;
+  }
+  return res.json();
+};
+
+export default async function MyAdvertsListPage() {
```

Comment:
I believe better naming for the route and page component is `/offers`
```suggestion
export default async function MyOffersPage() {
```

Reply:
Changed.